scripts/bert/staticbert/static_export_squad.py

```diff
@@ -0,0 +1,245 @@
+                         'options are book_corpus_wiki_en_uncased and book_corpus_wiki_en_cased.')
+
+parser.add_argument('--pretrained_bert_parameters',
+                    type=str,
+                    default=None,
+                    help='Pre-trained bert model parameter file. default is None')
+
+parser.add_argument('--uncased',
+                    action='store_false',
+                    help='if not set, inputs are converted to lower case.')
+
+parser.add_argument('--output_dir',
+                    type=str,
+                    default='./output_dir',
+                    help='The output directory where the model params will be written.'
+                         ' default is ./output_dir')
+
+parser.add_argument('--test_batch_size',
+                    type=int,
+                    default=24,
+                    help='Test batch size. default is 24')
+
+parser.add_argument('--max_seq_length',
+                    type=int,
+                    default=384,
+                    help='The maximum total input sequence length after WordPiece tokenization.'
+                         'Sequences longer than this will be truncated, and sequences shorter '
+                         'than this will be padded. default is 384')
+
+parser.add_argument('--doc_stride',
+                    type=int,
+                    default=128,
+                    help='When splitting up a long document into chunks, how much stride to '
+                         'take between chunks. default is 128')
+
+parser.add_argument('--max_query_length',
+                    type=int,
+                    default=64,
+                    help='The maximum number of tokens for the question. Questions longer than '
+                         'this will be truncated to this length. default is 64')
+
+parser.add_argument('--gpu', type=str, help='single gpu id')
+
+parser.add_argument('--seq_length',
+                    type=int,
+                    default=384,
+                    help='The sequence length of the input')
+
+parser.add_argument('--input_size',
+                    type=int,
+                    default=768,
+                    help='The embedding size of the input')
+
+parser.add_argument('--export',
+                    action='store_true',
+                    help='Whether to export the model.')
+
+parser.add_argument('--evaluate',
+                    action='store_true',
+                    help='Whether to evaluate the model.')
+
+args = parser.parse_args()
+
+
+###############################################################################
+#        Specify the static input size and sequence length                    #
+###############################################################################
+
+#SEQLENGTH stands for the sequence length of the input,
+#INPUTSIZE represents the embedding size of the input.
+os.environ['SEQLENGTH'] = str(args.seq_length)
+os.environ['INPUTSIZE'] = str(args.input_size)
```

Comment:
use command line options instead of environment variables. there's no need for more than one way of such input

Reply:
This is used in `StaticBaseTransformerEncoder`, command line options wouldn't work.

Reply:
I'm not sure I follow. IMO the class should take these as additional arguments.

Reply:
As I said, please take a look at `StaticBaseTransformerEncoder`, we are doing this 
`        length = int(os.environ['SEQLENGTH'])
        C_in = int(os.environ['INPUTSIZE'])` in the `hybrid_forward`. This is how static hybridizable BERT works. Otherwise, we need to modify the `StaticBERTModel` structure, then `get_bert_model` wouldn't work since it has different model architecture compared to pre-defined `BERTModel`.

Reply:
Thanks for the explanation. I suggest that we pass these arguments from the constructor of that block and update the static-BERT-specific `get_bert_model` method accordingly. Passing argument through an environment variable seems too hacky for the general users.

Reply:
Given that this is in the script and not in API, and the `export static bert` is an experimental API, and the current functionality doesn't affect the users. I would like to have this one for now. Besides, since I am not sure whether changing the model structure might affect parameter loading (when pretrained=True) in `get_static_bert_model` (this is from `BERTModel`), and this will generate more reductant codes such as pretrained settings, and might possibly involves more tests.

Reply:
> the current functionality doesn't affect the users

The scripts are to be used by regular users who want to try to deploy this model, so this is not true. Generally we don't check in code that won't be used.

> I am not sure whether changing the model structure might affect parameter loading

As long as you load the length from the parameter saved in the constructor in the places where it's reading from environment variables, the model structure is not really changed so it doesn't affect pre-trained parameter loading. Feel free to let me know if you need help on this.

> more reductant codes such as pretrained settings

Copying the pretrained settings shouldn't be necessary. They are just dictionaries that can be modified in the methods, and you can easily add new entries by `pretrained_parameters['seq_length'] = X`. It shouldn't require duplicating the parameters.

If you run into trouble feel free to ping me and I'm happy to offer help.

Reply:
Having a consistent design of the interface goes a long way in ensuring that things remain extensible and useful. The code that we write once will potentially be used and read by many people, for many times. Hope that you understand.

Reply:
>> the current functionality doesn't affect the users

>The scripts are to be used by regular users who want to try to deploy this model, so this is not true. Generally we don't check in code that won't be used.

Users won't need to write any code but just specify the `input_size` and `seq_length` in the command line, the usage of the `input_size` and `seq_length` in hiding in the API. It doesn't affect the users. 

>Having a consistent design of the interface goes a long way in ensuring that things remain extensible and useful. The code that we write once will potentially be used and read by many people, for many times. Hope that you understand.

I agree, that's the reason we have it now in the SCRIPT not API as an experimental API. If we would like to spend more time on this, I am also fine with it, just in terms of time, I need to spend more time on testing it out. Hope that you can understand that as well.

Reply:
I think you can skip rerunning the squad experiment. This is not a requirement of the export script.